citi-onboarding/karua-landing-page
# Hotfix/last changes

What I did

- Last changes

How to test

-  Open the terminal and go to the folder of this project

- In terminal:

git checkout develop
git pull
git checkout hotfix/lastChanges
git pull origin hotfix/lastChanges
yarn install
yarn start
In your browser go to http://localhost:3000/

Check if the component looks just like the mockup in different screen sizes
